hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/MigrationRecordsManager.java

```diff
@@ -139,8 +146,14 @@ public void publishMigrationRecords(final Instant now) {
         publishAccountsCreated(
                 systemAccountsCreator.getSystemAccountsCreated(), now, SYSTEM_ACCOUNT_CREATION_MEMO, "system creation");
 
+        if (bootstrapProperties.getBooleanProperty(PropertyNames.ACCOUNTS_BLOCKLIST_ENABLED)) {
+            publishAccountsCreated(
+                    blocklistAccountCreator.getBlockedAccountsCreated(), now, BLOCKED_ACCOUNT_CREATION_MEMO, "blocked");
```

Comment:
So these synthetic records _are_ able to be seen by querying a mirror node?  (If so, great!)